experimental/PropertyDDS/packages/property-inspector-table/package.json

```diff
@@ -120,11 +120,11 @@
 		"webpack-merge": "^5.8.0"
 	},
 	"peerDependencies": {
-		"@fluid-experimental/property-binder": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
-		"@fluid-experimental/property-changeset": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
-		"@fluid-experimental/property-dds": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
-		"@fluid-experimental/property-properties": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
-		"@fluid-experimental/property-proxy": ">=2.0.0-internal.3.2.0 <2.0.0-internal.4.0.0",
+		"@fluid-experimental/property-binder": "workspace:~",
+		"@fluid-experimental/property-changeset": "workspace:~",
+		"@fluid-experimental/property-dds": "workspace:~",
+		"@fluid-experimental/property-properties": "workspace:~",
+		"@fluid-experimental/property-proxy": "workspace:~",
```

Comment:
We should double-check that these get replaced properly in the release pipeline. We have had custom bump code in the past and it didn't handle peerdeps. Since we're no longer using custom version schemes pnpm itself should handle this and do the right thing, but we should test.

Reply:
Did some local testing and it worked as expected. I wasn't sure if Jason's change would handle this case but it does, so we're good.